.circleci/config.yml

```diff
@@ -74,12 +74,14 @@ jobs:
       - checkout
       - go/mod-download-cached
       - setup_remote_docker:
-          version: 17.07.0-ce
+          version: 18.06.0-ce
```

Comment:
Also you've forgotten to do the same for the `publish_release` job, let me do that